src/components/button/Button.tsx

```diff
@@ -1,21 +1,24 @@
 import * as React from 'react';
-import { View, Text, TouchableOpacity, TouchableNativeFeedback, GestureResponderEvent, Platform, AccessibilityTrait, ViewStyle } from 'react-native';
-
-const s = require('./Button.css');
+import { View, Text, TouchableOpacity, TouchableNativeFeedback, GestureResponderEvent,
+  Platform, AccessibilityTrait, ViewStyle, TextStyle } from 'react-native';
+import * as s from './Button.css';
 
 interface IProps {
   title: string;
+  testID?: string;
   accessibilityLabel?: string;
   disabled?: boolean;
   style?: ViewStyle;
   hasTVPreferredFocus?: boolean;
-  onPress?: (event: GestureResponderEvent) => void;
-  testID?: string;
+  onPress?(event: GestureResponderEvent): void;
 }
 
-export default class Button extends React.PureComponent<IProps> {
+// tslint:disable-next-line variable-name typedef
+const TouchableElement = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+
+export class Button extends React.PureComponent<IProps> {
 
-  render() {
+  public render(): JSX.Element {
```

Comment:
I'm not sure if it's not too strict as well. We have lot of components that are just a render function. In 99% this render function will return a `JSX.Element`.

And I'm scared we will result with lot of `void` on our own functions inside classes, because we don't want to type the return type of the function.

Reply:
Yes, sounds good!